lib/renderer/init.ts

```diff
@@ -150,7 +150,7 @@ if (cjsPreloads.length) {
   }
 }
 if (esmPreloads.length) {
-  const { runEntryPointWithESMLoader } = __non_webpack_require__('internal/modules/run_main');
+  const { runEntryPointWithESMLoader } = __non_webpack_require__('internal/modules/run_main') as typeof import('@node/lib/internal/modules/run_main');
```

Comment:
Can we not map this magically somehow in TS, declaring the type of __non_webpack_require__ better or something?